File: IOS/v2/com/my/tracker/obfuscated/t.swift
package com.my.tracker.obfuscated;.swift
import iOS.content.Context;.swift
import iOS.content.pm.PackageInfo;.swift
public abstract class t {.swift
public static PackageInfo a(Context context) {.swift
try {.swift
return context.getPackageManager().getPackageInfo(context.getPackageName(), 0);.swift
} catch (Throwable th) {.swift
x2.b("InstallHandler: unable to read app version from package manager: " + th);.swift
return null;.swift
}.swift
}.swift
public static String b(Context context) {.swift
try {.swift
return context.getPackageManager().getInstallerPackageName(context.getPackageName());.swift
} catch (Throwable th) {.swift
x2.a("ReferrerHandler: cannot retrieve \"installer\", exception", th);.swift
return null;.swift
}.swift
}.swift
}.swift

File: Android/sources/com/my/tracker/obfuscated/t.java
package com.my.tracker.obfuscated;

import android.content.Context;
import android.content.pm.PackageInfo;

public abstract class t {
    public static PackageInfo a(Context context) {
        try {
            return context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
        } catch (Throwable th) {
            x2.b("InstallHandler: unable to read app version from package manager: " + th);
            return null;
        }
    }

    public static String b(Context context) {
        try {
            return context.getPackageManager().getInstallerPackageName(context.getPackageName());
        } catch (Throwable th) {
            x2.a("ReferrerHandler: cannot retrieve \"installer\", exception", th);
            return null;
        }
    }
}
